### PR TITLE
New Cargo Crate - Mothroach

### PR DESCRIPTION
**Pull request by Miliviu**
## About The Pull Request

You can now buy a Crate with a Mothroach inside in cargo

## Why It's Good For The Game

They are cute and we need to buy some

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

image

</details>

## Changelog
:cl:
add: added the Mothroach cargo crate
/:cl:

### CODE REVIEW

**Comment by LodedDiper69** [comment]
This is the PR that will beestation 13 dot com

**Comment by Fronsis** [comment]
I see moth agenda being pushed i give thumbs up

File: code/modules/cargo/packs.dm [hunk 1]
```diff
@@ -2478,6 +2478,13 @@
 	contains = list(/mob/living/simple_animal/hostile/retaliate/goat)
 	crate_name = "goat crate"
 
+/datum/supply_pack/critter/mothroach
+	name = "Mothroach Crate"
+	desc = "Put the mothroach on your head and find out what true cuteness looks like."
+	cost = 1500
```
Comment (Crossedfall) [inline code/modules/cargo/packs.dm]: Feels a bit cheap. Maybe 5000 like the other pets?

Comment (Miliviu) [reply code/modules/cargo/packs.dm]: Fair enough!

**Comment by ValuedEmployee** [comment]
Ah, just what I needed!

**Review by PowerfulBacon — APPROVED** [review]

**Pull request merged**